bl-kernel/dbsite.class.php

```diff
@@ -110,7 +110,9 @@ public function autosaveInterval()
 
 	public function extremeFriendly()
 	{
-		return $this->getField('extremeFriendly');
+		//always true. value is string not bool. here i convert it to boolean
```

Comment:
When I store the data the variables are saved with the same type of the field defined, I don't know why you are casting to boolean.